ocdata/core/adsorbate.py

```diff
@@ -64,12 +64,7 @@ def __init__(
                 self.binding_indices = adsorbate_binding_indices
         elif adsorbate_id_from_db is not None:
             adsorbate_db = adsorbate_db or pickle.load(open(adsorbate_db_path, "rb"))
-            (
-                self.atoms,
-                self.smiles,
-                self.binding_indices,
-                self.reaction_string,
-            ) = adsorbate_db[adsorbate_id_from_db]
+            self._save_adsorbate(adsorbate_db[adsorbate_id_from_db])
```

Comment:
nit: `_load_adsorbate` instead of `_save_adsorbate` maybe?

Reply:
done